Project.toml

```diff
@@ -44,7 +44,7 @@ Reactant = "3c362404-f566-11ee-1572-e11a4b42c853"
 [extensions]
 OceananigansEnzymeExt = "Enzyme"
 OceananigansMakieExt = ["MakieCore", "Makie"]
-OceananigansReactantExt = ["Reactant", "KernelAbstractions"]
+OceananigansReactantExt = "Reactant"
```

Comment:
This is wrong, this undoes  #4085

Reply:
Ok, we can put in a PR to add it back but the problem is that all tests fail, so we cannot develop further with this change.